spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java

```diff
@@ -559,18 +559,18 @@ public void setMicrometerTags(Map<String, String> tags) {
 		return Collections.unmodifiableMap(this.micrometerTags);
 	}
 
-	public Duration getConsumerStartTimout() {
-		return this.consumerStartTimout;
+	public Duration getConsumerStartTimeout() {
```

Comment:
We can't do this unconditionally: it is a breaking change.
Even if it is a bug, some projects may rely on calling this API, therefore changing names for these method will cause a compilation (and execution) problem in those target applications.

We may consider to accept the change as is for the current `2.7` version, but we definitely should consider to have a `@Deprecated` variant for the previous `2.6.x` version. Although I would stay with deprecated variants until the next minor (or major) version...

Reply:
You're right. I restored the methods and marked them deprecated